keps/349-all-or-nothing/README.md

```diff
@@ -215,6 +218,15 @@ type WaitForPodsReady struct {
 	// enforced only if waitForPodsReady.enable=true. If unspecified, it defaults to 5min.
 	// +optional
 	TimeoutSeconds *int64 `json:"timeoutSeconds,omitempty"`
+
+	// BlockAdmission when true, cluster queue will block admissions for all
+	// subsequent jobs until the jobs reach the PodsReady=true condition.
+	// This setting is only honored when `Enable` is set to true.
+	BlockAdmission *bool `json:"blockAdmission,omitempty"`
+
+	// RequeuingStrategy defines the strategy for requeuing a Workload.
+	// +optional
+	RequeuingStrategy *RequeuingStrategy `json:"requeuingStrategy,omitempty"`
```

Comment:
Could you also include the `RequeuingStrategy` type?

Reply:
Absolutely, done!